### ISSUE TITLE
Sitemap has wrong home URL when a site has multiple domains (for different languages)

### ISSUE THREAD

**Issue by tnottu** [sybrew/the-seo-framework#325]
I am using Polylang Pro -plugin to create multiple languages for a WordPress-site (not multi-site / network).  Each language has it's own domain, e.g.: `website.fi`, `website.se`, `website.de`. The sitemap.xml (generated by The SEO Framework) has correct URLs for all pages and articles, but the home URL always points to the main language, in this case, Finnish.

For example, from the sitemap for Swedish:

```
<url>
    <loc>https://website.fi/</loc>  <!-- Incorrect: Home URL for Finnish version -->
    <lastmod>2018-01-17T09:20+02:00</lastmod>
    <priority>1.0</priority>
</url>
<url>
    <loc>https://website.se/nyheter/</loc>  <!-- Correct: News page for Swedish version -->
    <lastmod>2018-04-25T16:56+03:00</lastmod>
    <priority>0.9</priority>
</url>
```

Is this something that can be fixed in The SEO Framework -plugin, or a problem with Polylang?

**Comment by alapiere** [comment]
I almost broke my international website SEO due to this canonical bug .. Hopefully I saw it and could force canonical on secondary language in homepage settings .. but it's not safe. Where could this bug be fixed ? polylang side or seo framework side ?

**Comment by sybrew** [comment]
Hi @alapiere 

If you wish to fix this, then I recommend setting a custom canonical URL for the affected page which points to the real URL.

You can do this under the Visibility tab of the page. Unfortunately, you'll have to maintain this manually.

I'm looking into a fix for the next major version, as we haven't heard back from Polylang's developers for a semantic resolution.

**Comment by sybrew** [comment]
I believe this has been resolved as of 3.2.4 in fd635c25e0c44e6c1765c1db95cc0c4864e9257a.

As such, I'm closing this issue. If, however, you still find this issue to be present, feel free to respond to this issue.
